python/heterocl/devices.py

```diff
@@ -176,8 +191,11 @@ class CPU(Device):
     def __init__(self, vendor, model, **kwargs):
         if vendor not in ["riscv", "arm", "intel", "sparc", "powerpc"]: 
             raise DeviceError(vendor + " not supported yet")
-        assert "cpu_" + model in model_table[vendor], \
-            model + " not supported yet"
+        if model is not None:
+            assert "cpu_" + model in model_table[vendor], \
+                model + " not supported yet"
+        else:
+            model = model_table[vendor][0]
```

Comment:
Maybe we can rearrage the `model_table` to be two-level? So we don't need to hard-code the `vendor not in ....` line. For examples, we should use `if vendor not in model_table['cpu']`.